pkg/sql/sem/builtins/builtins.go

```diff
@@ -7354,6 +7359,158 @@ expires until the statement bundle is collected`,
 			Volatility: volatility.Volatile,
 		},
 	),
+	"crdb_internal.fingerprint": makeBuiltin(
```

Comment:
We may want to check the `V23_1` version gate here and error out, since we'll otherwise effectively be sending export requests without the `ExportFingerprint` parameter set and run actual exports, storing the SSTs in memory, which can OOM the node.

Reply:
good point, done.